view/frontend/web/internals/common.js

```diff
@@ -45,6 +45,23 @@ var algolia = {
 		}, originalData);
 
 		return data;
+	},
+	htmlspecialcharsDecode: function(string) {
+		var unescapedString = string,
+			specialchars = [
+				[ '"', '&quot;' ],
+				[ '>', '&gt;' ],
+				[ '<', '&lt;' ],
+				[ '&', '&amp;' ],
+				[ "'", '&#39;' ]
+			];
```

Comment:
Hey team,

The OWASP recommends to escape forward slash too (https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html). Not sure how necessary it is to actively prevent XSS, but I think it's still worth including.